packages/mui-joy/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,180 @@
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+import { OverridableComponent } from '@mui/types';
+import { unstable_capitalize as capitalize } from '@mui/utils';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { useThemeProps } from '../styles';
+import styled from '../styles/styled';
+import breadcrumbsClasses, { getBreadcrumbsUtilityClass } from './breadcrumbsClasses';
+import { BreadcrumbsProps, BreadcrumbsTypeMap } from './BreadcrumbsProps';
+
+const useUtilityClasses = (ownerState: BreadcrumbsProps) => {
+  const { size } = ownerState;
+
+  const slots = {
+    root: ['root', size && `size${capitalize(size)}`],
+    li: ['li'],
+    ol: ['ol'],
+    separator: ['separator'],
+  };
+
+  return composeClasses(slots, getBreadcrumbsUtilityClass, {});
+};
+
+const BreadcrumbsRoot = styled('nav', {
+  name: 'MuiBreadcrumbs',
+  slot: 'Root',
+  overridesResolver: (props, styles) => {
+    return [{ [`& .${breadcrumbsClasses.li}`]: styles.li }, styles.root];
+  },
+})<{ ownerState: BreadcrumbsProps }>(({ theme, ownerState }) => {
+  return [
```

Comment:
No need to use an array in this case.

```suggestion
  ({
```